README.md

```diff
@@ -66,6 +66,22 @@ For example:
 
 will clean and restore volume named `some_volume` from `/tmp/some_archive.tar.bz2` archive file.
 
+### Copy volume between hosts
+
+One good example of how you can use the output to stdout is directly migrating the volume to a new host
+
+Syntax:
+
+    docker run -v [volume-name]:/volume --rm loomchild/volume-backup backup - |\
+         ssh [receiver] docker run -i -v [volume-name]:/volume --rm loomchild/volume-backup restore -
+
+*Note* In case traffic is not an issue it is recommended to turn of compression for this process
```

Comment:
Could you clarify why do you recommend disabling the compression?

Reply:
As I mentioned in the line if traffic is of no concern I would highly recommend turning off compression to save on cpu cycles and massively speed up the transfer. Especially bzip2 is relatively cpu intensive and the transfer took 2-3x as long on my old machine I was transfering away from.